contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -212,13 +212,20 @@ contract ValidatorShare is IValidatorShare {
     function getLiquidRewards(address user)
         public
         view
-        returns (uint256 liquidRewards)
+        returns (uint256)
     {
         uint256 share = balanceOf(user);
-        uint256 _exchangeRate = exchangeRate();
-        require(share > 0, "Zero balance");
-        uint256 totalTokens = _exchangeRate.mul(share).div(100);
-        liquidRewards = totalTokens.sub(amountStaked[user]);
+        if (share == 0) {
+            return 0;
+        }
+        
+        uint256 liquidRewards;
+        uint256 totalTokens = exchangeRate().mul(share).div(100);
+        if (totalTokens >= amountStaked[user]) {
+            liquidRewards = totalTokens.sub(amountStaked[user]);
+        }
+
+        return liquidRewards;
```

Comment:
Less code -> easier on the eyes and saved gas.
```suggestion
        uint256 totalTokens = exchangeRate().mul(share).div(100);
        if (totalTokens >= amountStaked[user]) {
            return totalTokens.sub(amountStaked[user]);
        }
        return 0;
```

Reply:
I would argue that your version uses less gas. With early exit i don't read any of the storage variables. 32 bytes of memory expansion cost 1 gas, so in general my implementation is more efficient, i only need to read amountStaked[user] once.

Reply:
Only case where your version requires less gas is when shares == 0. I personally like to optimize for the usual case, not an outlier.